test: Remove `mock-workflows` from deployments acceptance tests

alfrunes opened the pull request.

alfrunes committed:
test: Remove `mock-workflows` from deployments acceptance tests

Signed-off-by: Alf-Rune Siqveland <alf.rune@northern.tech>

merlin-northern commented on backend/services/deployments/tests/tests/conftest.py:
```diff
@@ -0,0 +1,38 @@
+    def __getattribute__(self, name: str, /) -> Any:
+        if name.startswith("do_"):
+            return self._handle_request
+        return super().__getattribute__(name)
```
could you leave some comment about this method, I have no idea why it is needed here.

alfrunes replied:
I added a docstring for the method.

alfrunes committed:
chore: Cleanup unused `.github` directories from old repositories

Signed-off-by: Alf-Rune Siqveland <alf.rune@northern.tech>

alfrunes committed:
ci: Remove deployments "mock-workflows" from dependabot

Signed-off-by: Alf-Rune Siqveland <alf.rune@northern.tech>

alfrunes commented:
I added a docstring and removed the dependabot tracking for the service, and also discovering that we have some dangling `.github` directories from the old repos still left over that I cleaned up :broom:
